spec/recursive_open_struct_spec.rb

```diff
@@ -116,6 +121,9 @@
 
       context "when recursing over arrays is enabled" do
         subject { RecursiveOpenStruct.new(h, :recurse_over_arrays => true) }
+        let(:arrays) { RecursiveOpenStruct.new(h, :arrays => true) }
+
+        it { subject.blah[0].should eq arrays.blah[0] }
```

Comment:
This should be tested in its own context or describe block (could be within `when cursing over arrays is enabled`) that explains that the `:arrays` shortcut is an alias for (or rather, requivalent to) `:recurse_over_arrays`.